patients/patients.go

```diff
@@ -44,6 +44,7 @@ type Service interface {
 	List(ctx context.Context, filter *Filter, pagination store.Pagination, sort []*store.Sort) (*ListResult, error)
 	Create(ctx context.Context, patient Patient) (*Patient, error)
 	Update(ctx context.Context, update PatientUpdate) (*Patient, error)
+	UpdateLastReviewed(ctx context.Context, update PatientUpdate) (*Patient, error)
```

Comment:
Updates of the patient object should use the `update` method of the service. It even has the exact same signature as this one.

Reply:
moved away from full patient update, just updating the single review field now.